migrations/founders.kovan.json

```diff
@@ -0,0 +1,14 @@
+{
+  "founders": [
```

Comment:
please add a comment that this is for testing purpose only .

Reply:
why do we need to forge with founders for kovan ?Is that only for test ? 
Is that for alchemy on Kovan ? If yes, does some one has the prv keys for these  addresses ?

Reply:
this is why I dont think this file should be in the repo, we should be able to pass in a founders file if we want

Reply:
yes this is to setup the genesis DAO how we want it setup on deployment

Reply:
I agree with @tibetsprague  I feel that the genesisParams + the genesis_test founders + genesis dao should not be part of the arc.js .there should be passed as params via alchemy ...(manually or via file).

@tibetsprague I guess that this setup is not for the real final genesis DAO . it is just for demo. 
and it should be comment and maybe also should be reflected in the dao name.

Reply:
hmm, yeah I guess we could deploy without any DAOs and setup the Genesis DAO manually. but its nice to have it setup for us too for dev and testing

Reply:
"but its nice to have it setup for us too for dev and testing" 
Yes , but you can have it by default on alchemy (not arc.js)..no ?

Reply:
how so? deployment happens in Arc.js. I mean its nice having a DAO setup on deployment, so we dont have to do it manually each time.